### DISCUSSION TITLE
Identify nx managed dependencies

### DISCUSSION THREAD

**Discussion by ijaouani** [nrwl/nx#18790] [General]
Hello,

We are using nx since multiple years and we love it. There is one thing I am looking for a while is the ability to identify **nx managed dependencies**.

Why ? 

When working with nx, package.json always result with a mix of :

1. nx deps (added when creating the workplace or adding nx plugin for example) 
2. other manually added deps with `npm install` (which have nothing related to nx).

So at then end of the day we never know when doing `npm outdated` which upgrades should be managed on our own and which one should be untouched because already managed by nx (with `nx mgirate`)

Thanks for any ideas.

**Comment by JoshMentzer** [comment]
I'd add on to this that it would be kind of neat to see some sort of migration report or dep-graph type view of non-managed dependencies affected by an update.  Like nx migrate building a little .md of 'here's what we updated, and here's your packages that may need to be manually addressed.'
